docs/dietpi_tools.md

```diff
@@ -691,14 +690,64 @@ dietpi-backup
 
 ![DietPi-Backup menu screenshot](assets/images/dietpi-backup_1.png){: width="681" height="330" loading="lazy"}
 
-!!! info "DietPi userdata may not be included"
+=== "Automatic daily backup"
 
-    If DietPi userdata have been moved to an external drive, i.e. `/mnt/dietpi_userdata` is a symlink, its content is excluded from backup and restore by default. You can change this with the `Filter` option.
+    `Dietpi-Backup` gives the option of an automatic daily backup function (controlled via the Linux `cron` mechanism).
+
+    It contains these options (see screenshot above):
+
+    - "Daily Backup": Activates the daily backup
+    - "Amount": Sets the number of backups to be kept. Backups are rotated automatically, if the maximum amount has been reached, the oldest backup is used as basis for the incremental new backup sync
+
+=== "Daily backup execution time"
+
+    The automatic daily backup (activated via option "Daily Backup", see screenshot above) is controlled via the Linux `cron` mechanism. Setting a different starting time can be an option, e.g. if you have several backup clients backing up to the same storage (backup server): Shifting the backup starting time of these systems may reduce temporary overload of the backup server by avoiding concurrent access to the storage.
+
+    The starting time is basically defined via the file `/etc/crontab` (which calls the backup/restore function via the `/etc/cron.daily/dietpi` script). It can be changed via the entry `cron.daily` within [`dietpi-cron`](#dietpi-cron). It is executed by running the following command
+
+    ```sh
+    dietpi-cron
+    ```
```

Comment:
Probably this fits better into the "Automatic daily backup" tab, since both belong together?

Reply:
Ok.